src/Button.tsx

```diff
@@ -6,53 +6,57 @@ import { FunctionComponent } from 'react';
 import { Box, IBox, IBoxCSS } from './Box';
 import { useTheme } from './theme';
 
-export const Button: FunctionComponent<IButton> = props => {
-  const { as = 'button', disabled, ...rest } = props;
-
-  const css = buttonStyles({ disabled });
-
-  return <Box {...rest} as={as} css={css} disabled={disabled} />;
-};
+// TODO better active styling
 
-export interface IButton extends IButtonProps, IBox<HTMLButtonElement> {}
-
-export interface IButtonProps {
+export interface IButton extends IBox<HTMLButtonElement> {
   disabled?: boolean;
 }
 
-export const buttonStyles = ({ disabled }: IButtonProps = {}): IBoxCSS => {
-  const theme = useTheme();
+export const Button: FunctionComponent<IButton> = props => {
+  const { as = 'button', ...rest } = props;
+  const css = buttonStyles(props);
+
+  return <Box {...rest} as={as} css={css} />;
+};
+
+export const buttonStyles = ({ disabled }: IButton = {}): IBoxCSS => {
+  const { button } = useTheme();
 
   return [
     {
+      color: button.fg,
+      backgroundColor: button.bg,
+      borderColor: button.border,
+
+      padding: '5px 10px',
```

Comment:
This is a breaking change.
I used the button component on different occasions. Plenty of my buttons were very small, i.e they had only an icon inside.

Reply:
Updated the above image to include a button with an icon. Let me know what you think

Reply:
It looks very nice, but I am afraid it's just too big :(
Could we introduce `size` prop or something similar?

Reply:
@P0lip I think having a size prop for all of our components would be ideal and something we should for sure add at some point. Can you show me or include an image where this would currently be breaking? I would like to get this repo and pr moving so we can start use the components practically. 

Unfortunately since we aren't using the components meaningfully (right now they are just slapped in as replacements in studio) its hard to tell what needs to be improved upon.

Reply:
You can take a look at Slate's Toolbar, for instance. It has fairly small buttons.